packages/README.md

```diff
@@ -0,0 +1,41 @@
+Packages
+================
+
+A package allows the creation of modular code that cen easily be shared. For example, packages
+can enable use cases to make network requests, integrate with device APIs, and so on. The minimal
+components of a package include:
+
+* **`pubspec.yaml`** A metadata file declaring package name, version, author, etc.
+
+* **`lib`** The lib directory containing the package's public code and minimally contains a single `<package-name>.dart` file.
+
+To implement a pure Dart package, you can either add the functionality inside `lib/<package name>.dart` or add 
+the functionality in several files inside the `lib` directory. 
+
+This directory contains several examples of packages used with Flutter. 
+For more information on how to use or develop new packages, see the [Using Packages](https://flutter.dev/docs/development/packages-and-plugins/using-packages/) guide.
+
+Shared Packages
+===============================
+Flutter supports users sharing their developed packages to the Flutter and Dart community. 
+
+Flutter packages are published to the ['pub.dev'](https://pub.dev/) page. Users can find the most downloaded
```

Comment:
at line 22, `['pub.dev'](https://pub.dev/) page.` should be `['pub.dev'](https://pub.dev/) website.`

Reply:
updated text